backend/src/sentiment/sentiment.py

```diff
@@ -0,0 +1,31 @@
+from typing import List, Dict
+import spacy
+from spacytextblob.spacytextblob import SpacyTextBlob
+
+# Takes ~1 min to run on 100 reviews.
+def sentiment(reviews: List[Dict]) -> Dict:
+    if len(reviews) == 0:
+        return "Product has no reviews."
+
+    sentiments = list(map(analyze, reviews))
+    positive_index = sentiments.index(max(sentiments))
+    negative_index = sentiments.index(min(sentiments))
+    
+    return {
+        "top_positive": {
+            "title": reviews[positive_index].get("title"),
+            "text": reviews[positive_index].get("comment"),
+            "rating": reviews[positive_index].get("rating"),
+        },
+        "top_negative": {
+            "title": reviews[negative_index].get("title"),
+            "text": reviews[negative_index].get("comment"),
+            "rating": reviews[negative_index].get("rating"),
+        }
+    }
+
+# TODO: If sentiment analysis becomes too slow, we can only analyze reviews with 1 or 5 star ratings.
```

Comment:
Are the 100 reviews being provided all explicitly 1 or 5 stars right now? Or just a combination of 100 reviews at any star?

It might be a good idea to limit it to 1 and 5 star reviews, if not already. Around 1min for 100 reviews seems a little long (that'd be 3min to see the comparison table just from the most positive/negative reviews part).